### PR TITLE
Upgrade each vendor to latest and replace `github.com/googleapis/gnostic` with `github.com/google/gnostic`

### PR DESCRIPTION
**Pull request by zchee**
Upgrade each vendor to the latest, also replace `github.com/googleapis/gnostic` with `github.com/google/gnostic` for https://github.com/google/gnostic/pull/259.

### CODE REVIEW

**Comment by zchee** [comment]
/cc @timburks 

depends on #72, I'll rebase after merged that.

File: go.mod [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
 )
+
+require (
+	golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd // indirect
+	golang.org/x/text v0.3.0 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+)
+
+// until google/gnostic release new version
+replace github.com/google/gnostic v0.5.5 => github.com/google/gnostic v0.5.6-0.20210930170106-c7a5e4fe8b37
```
Comment (zchee) [inline go.mod:26]: I'll revert after google/gnostic released a new version.

**Comment by zchee** [comment]
@LorenzHW done to rebase.
PTAL to https://github.com/google/gnostic-grpc/pull/73#discussion_r725650349

**Comment by zchee** [comment]
Hmm, is this CI results are no problem? (I'd temporary upgrade go version to go1.17
happened below error:

```

# github.com/google/gnostic-grpc/incompatibility
20
incompatibility/apiset.go:31:21: undefined: ApiSetIncompatibility
21
incompatibility/apiset.go:48:37: undefined: ApiSetIncompatibility
22
incompatibility/apiset.go:60:27: undefined: IncompatibilityReport
23
incompatibility/apiset.go:60:51: undefined: ApiSetIncompatibility
24
incompatibility/apiset.go:80:35: undefined: IncompatibilityReport
25
incompatibility/apiset.go:80:59: undefined: ApiSetIncompatibility
26
incompatibility/apiset.go:89:45: undefined: IncompatibilityAnalysis
27
incompatibility/apiset.go:104:46: undefined: FileIncompatibilityClassificationAnalysis
28
incompatibility/apiset.go:111:51: undefined: Incompatibility
29
incompatibility/incomp-scanning.go:83:42: undefined: IncompatibilityReport
30
incompatibility/incomp-scanning.go:83:42: too many errors
```

https://github.com/google/gnostic-grpc/pull/73/checks?check_run_id=3851957059

File: go.mod (hunk 1 above)
Comment (LorenzHW) [reply go.mod:26]: Works for me.

**Comment by LorenzHW** [comment]
> Hmm, is this CI results are no problem? (I'd temporary upgrade go version to go1.17 happened below error:
> 
> ```
> 
> # github.com/google/gnostic-grpc/incompatibility
> 20
> incompatibility/apiset.go:31:21: undefined: ApiSetIncompatibility
> 21
> incompatibility/apiset.go:48:37: undefined: ApiSetIncompatibility
> 22
> incompatibility/apiset.go:60:27: undefined: IncompatibilityReport
> 23
> incompatibility/apiset.go:60:51: undefined: ApiSetIncompatibility
> 24
> incompatibility/apiset.go:80:35: undefined: IncompatibilityReport
> 25
> incompatibility/apiset.go:80:59: undefined: ApiSetIncompatibility
> 26
> incompatibility/apiset.go:89:45: undefined: IncompatibilityAnalysis
> 27
> incompatibility/apiset.go:104:46: undefined: FileIncompatibilityClassificationAnalysis
> 28
> incompatibility/apiset.go:111:51: undefined: Incompatibility
> 29
> incompatibility/incomp-scanning.go:83:42: undefined: IncompatibilityReport
> 30
> incompatibility/incomp-scanning.go:83:42: too many errors
> ```
> 
> https://github.com/google/gnostic-grpc/pull/73/checks?check_run_id=3851957059

Hmm, I am not entirely sure why this is happening. Shouldn't tests be red if it is a problem?

**Comment by zchee** [comment]
@LorenzHW 
> Hmm, I am not entirely sure why this is happening. Shouldn't tests be red if it is a problem?


OK, I'll dig it. wait a moment, pls.

File: go.mod (hunk 1 above)
Comment (timburks) [reply go.mod:26]: I just pushed https://github.com/google/gnostic/releases/tag/v0.5.6 - @zchee will that help?

Comment (zchee) [reply go.mod:26]: @timburks Okay, I'll send PR soon.

**Pull request merged**